apps/server/src/app.ts

```diff
@@ -16,27 +16,10 @@ const app = express();
 
 const port = ENV.PORT || 5500;
 
-const allowedOrigins: Array<string> = [
-  ENV.FE_BASE_URL as string,
-  // CORS allow use of swagger on local environment
-  ENV.IS_PROD ? "" : `http://localhost:${port}`,
-].filter(Boolean);
 
-const corsOptions: cors.CorsOptions = {
-  origin: allowedOrigins,
-  methods: ["GET", "POST", "PUT", "DELETE", "OPTIONS"],
-  credentials: true,
-};
```

Comment:
Why did you delete these cors methods??

Reply:
It was redundant, not allowing me to access another page